automated install/basic-install.sh

```diff
@@ -220,6 +220,19 @@ getGitFiles() {
   return 0
 }
 
+resetRepo() {
+  local directory="${1}"
+  local curdir
+
+  curdir="${PWD}"
```

Comment:
This is fine, but since this happens in a function, there's really no need to get the PWD and change back, the function will change to the `git` inside a subshell and not affect the flow of the caller script.